src/Cake/project.json

```diff
@@ -1,8 +1,9 @@
 ﻿{
-  "version": "0.16.1-*",
+  "version": "0.16.2-*",
```

Comment:
Think that'll be patched by build.cake&GitVersion during build. As AppVeyor PR artifacts all  v0.17.0 both nuget & assemblies, hopefully hotfix branch will get correct versioning once PR merged :)